django_pain/management/commands/import_payments.py

```diff
@@ -0,0 +1,64 @@
+"""Command for importing payments from bank."""
+import importlib
+import sys
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from django_pain.models import BankAccount
+
+
+class Command(BaseCommand):
+    """Import payments from bank."""
+
+    help = 'Import payments from the bank. Bank statement should be provided on standard input.'
+
+    def add_arguments(self, parser):
+        """Command takes one argument - dotted path to parser class."""
+        parser.add_argument('parser', type=str, help='dotted path to parser class')
```

Comment:
I find this usage rather cumbersome. If the statement is in the file on drive, it is required to be passed as standard input. Also prevents a simple possibility to batch import several statements at once.

Instead of
```
cat statement.xml | django-admin import_payments parsers.fio_xml.FioXMLParser
```
I suggest
```
django-admin import_payments -p parsers.fio_xml.FioXMLParser statement.xml
```
(usage `django-admin import_payments [options] -p <parser> [<statement>, ...]`)

`-p` option may become optional, if we provide reasonable autodetection.